tools/osbuilder/rootfs-builder/debian/Dockerfile-aarch64.in

```diff
@@ -8,7 +8,7 @@ ARG IMAGE_REGISTRY=docker.io
 FROM ${IMAGE_REGISTRY}/debian:@OS_VERSION@
 
 # RUN commands
-RUN apt-get update && apt-get install -y \
+RUN apt-get update && apt-get install -y --no-install-recommends \
```

Comment:
Non-advice. Just want to express that this has taught me a lot :)

Reply:
Me either!